HACS compatibility

hectorespert opened the pull request:
Modify project to support HACS: https://github.com/SpuelMett/Boinc-Home-Assistant-Integration/issues/6

hectorespert committed:
Support HACS structure

hectorespert committed:
Add hassfest workflow

hectorespert committed:
Ignored checks

hectorespert committed:
Delete hassfest

hectorespert committed:
Fix ignored

SpuelMett commented:
Thank you for the effort so far. I think to fulfill the requirements for an integration, a `Hassfest` Github workflow is also needed.  As seen [here](https://www.hacs.xyz/docs/publish/include/#before-you-start).
 
 Furthermore i think that the README also needs a [My Link](https://www.hacs.xyz/docs/publish/start/#my-links) like this:
 
 [![Open your Home Assistant instance and open a repository inside the Home Assistant Community Store.](https://my.home-assistant.io/badges/hacs_repository.svg)](https://my.home-assistant.io/redirect/hacs_repository/?owner=SpuelMett&repository=Boinc-Home-Assistant-Integration)
 
 Correct me if i am wrong here.

hectorespert committed:
Merge branch 'main' into main

hectorespert committed:
Add hassfest workflow

hectorespert committed:
Fix workflows

hectorespert commented:
@SpuelMett I added the hassfest workflow and reverted the changes in the README.md

I’d recommend merging these changes so the workflows can run, and then fixing any issues that come up afterward.

hectorespert committed:
Remove ignored checks
